### PR TITLE
Fixes #21136:  Minify js produced by elm - plugins

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/21136

### CODE REVIEW

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

File: plugins-common/elm.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  echo "$ curl -L -o ${ELM}.gz https://github.com/elm/compiler/releases/download/${ELM}/binary-for-linux-64-bit.gz"
+  echo "$ gzip -d ${ELM}.gz"
+  echo "$ chmod +x ${ELM}"
+  echo "# then put it somewhere in your PATH"
+  exit 1
+fi
+
+build_release() {
+  ${ELM} make --optimize sources/${app}.elm --output=generated/rudder-${app_l}.js
+  terser generated/rudder-${app_l}.js --compress 'pure_funcs="F2,F3,F4,F5,F6,F7,F8,F9,A2,A3,A4,A5,A6,A7,A8,A9",pure_getters,keep_fargs=false,unsafe_comps,unsafe' | terser --mangle --output=generated/rudder-${app_l}.min.js
```
Comment (ncharles) [inline plugins-common/elm.sh:39]: this includes a dependency to terser that needs to be documented

**Review by VinceMacBuche — APPROVED** [review]

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**